dictionary/MessageFormatDictionary.js

```diff
@@ -80,22 +80,54 @@ export default class MessageFormatDictionary extends Dictionary {
 	 *        Defaults to an empty plain object.
 	 */
 	get(key, parameters = {}) {
+		let scope = this._getScope(key);
+
+		if (scope === false) {
+			throw new GenericError(
+				`ima.dictionary.MessageFormatDictionary.get: The ` +
+				`localization phrase '${key}' does not exists`,
+				{ key, parameters }
+			);
+		}
+
+		return scope(parameters);
+	}
+
+	/**
+	 * @inheritdoc
+	 * @param {string} key The key identifying the localization phrase. The key
+	 *        consists of at least two parts separated by dots. The first part
+	 *        denotes the name of the source JSON localization file, while the
+	 *        rest denote a field path within the localization object within
+	 *        the given localization file.
+	 */
+	has(key) {
+		let scope = this._getScope(key);
+		return (scope === false) ? false : true;
```

Comment:
The whole body of this method can (and should) be simplified in the following fashion:
`return !!this._getScope(key);`